docs/core/versions/index.md

```diff
@@ -13,89 +13,220 @@ ms.assetid: f6f684b1-1d2c-4105-8376-7c1959e23803
+The installer UI in those downloads will also be modified to clearly present the names and versions of the components being installed. In particular, titles will show the same version number that is in the download's file name.
+
+#### File name format
+
+`[product]-[component]-[major].[minor].[patch]-[previewN]-[optional build #]-[rid].[file ext]`
+
+Here are some examples of this format:
+
+```
+dotnet-runtime-2.0.4-macos.10.12-x64.pkg            # Mac runtime installer
+dotnet-sdk-2.0.4-win10-x64.exe                      # Windows SDK installer
+dotnet-sdk-2.0.4-fedora.24-x64.tar.gz               # Fedora 24 binary archive
+
+#Ubuntu file set needed for the SDK
+dotnet-host-2.0.4-ubuntu.16.04-x64.deb              # Host / muxer and host policy
+dotnet-runtime-2.0.4-ubuntu.16.04-x64.deb           # runtime
+dotnet-sdk-2.0.4-ubuntu.16.04-x64.deb               # SDK tools
+```
+
+The format is readable and clearly shows what you're downloading, what version it is, and where you can use it. The runtime package name includes `runtime`, and the SDK includes `SDK`.
+
+#### UI string format
+
+All web site descriptions and UI strings in the installers are kept consistent, accurate, and simple. Following are some examples of this.
+
+| Installer | Window Title | Other content in installer | What is installed |
+| :--       | :--          | :--           | :--               |
+| SDK       | .NET Core 2.0 SDK (x64) Installer | .NET Core 2.0.4 SDK  | .NET Core 2.0.4 Tools + .NET Core 2.0.4 Runtime |
+| Runtime   | .NET Core 2.0 Runtime (x64) Installer | .NET Core 2.0.4 Runtime | .NET Core 2.0.4 Runtime |
+
+Preview releases will differ only slighty:
+
+| Installer | Window Title | Other content in installer | What is installed |
+| :--       | :--          | :--           | :--               |
+| SDK       | .NET Core 2.0 Preview 1 SDK (x64) Installer | .NET Core 2.0.0 Preview 1 SDK  | .NET Core 2.0.0 Preview 1 Tools + .NET Core 2.0.0 Preview 1 Runtime |
+| Runtime   | .NET Core 2.0 Preview 1 Runtime (x64) Installer | .NET Core 2.0.0 Preview 1 Runtime | .NET Core 2.0.0 Preview 1 Runtime |
+
+It may happen that an SDK release contains more than one version of the runtime. When that happens, the installer UX will look like the following (only the SDK is shown since the Runtime is irrelevant):
+
+| Installer | Window Title | Other content in installer | What is installed |
+| :--       | :--          | :--           | :--               |
+| SDK       | .NET Core 2.1 SDK (x64) Installer | .NET Core 2.1.1 SDK and Runtime <br> .NET Core 2.0.6 Runtime | .NET Core 2.1.1 Tools + .NET Core 2.1.1 Runtime + .NET Core 2.0.6 Runtime|
+
+It is also possible that .NET Core Tools need to be updated, without runtime changes. In that case, we would bump the SDK version (for example, to 2.1.2) and then have the Runtime catch up the next time it shipped (for example, both the Runtime and SDK ship the following time as 2.1.3).
+
+### Package managers
```

Comment:
This might be one of those key scenarios to mention when describing scenarios where these details are important.